interfaces/builtin/core_support.go

```diff
@@ -50,6 +50,10 @@ const coreSupportConnectedPlugAppArmor = `
 /{,usr/}{,s}bin/sysctl                 ixr,
 @{PROC}/sys/{,**}                      r,
 @{PROC}/sys/**                         w,
+
+# Allow modifying logind configuration from the core snap
+/etc/systemd/logind.conf.d/{,*}         r,
+/etc/systemd/logind.conf.d/{,core.conf} w,
```

Comment:
FYI, it looks like /etc/systemd/logind.conf.d is in /etc/system-image/writable-paths, but it doesn't exist. I'll assume this directory will be created via some other mechanism then. Your rules are written such that creating the directory is allowed but as /etc/systemd is readonly, it won't work. I'm not sure what your intent is here, so I'll assume that something else has created /etc/systemd/logind.conf.d/ and that it is writable and go from there.

I'd prefer to use the pattern we used for rsyslog and sysctl and use this rule:
```
# Allow modifying logind configuration. For now, allow reading all logind
# configuration but only allow modifying NN-snap*.conf and snap*.conf files
# in /etc/systemd/logind.conf.d.
/etc/systemd/logind.conf                            r,
/etc/systemd/logind.conf.d/{,*}                     r,
/etc/systemd/logind.conf.d/{,[0-9][0-9]-}snap*.conf w,
```

This gives you the flexibility to create any number of files but they are still namespaced for snappy. Note that from https://www.freedesktop.org/software/systemd/man/logind.conf.html: "It is recommended to prefix all filenames in those subdirectories with a two-digit number and a dash, to simplify the ordering of the files."

The rules I gave don't enforce that, but allow it.